job-dsl-plugin/build.gradle

```diff
@@ -22,9 +22,7 @@ jenkinsPlugin {
     displayName = 'Job DSL'
     url = 'https://wiki.jenkins-ci.org/display/JENKINS/Job+DSL+Plugin'
     gitHubUrl = 'https://github.com/jenkinsci/job-dsl-plugin'
-    maskClasses = 'groovy.grape'
     compatibleSinceVersion = '1.35'
-    pluginFirstClassLoader = true
```

Comment:
Do we need to adjust the guava version to match the one from Jenkins core? Does the groovy version already match the one from Jenkins core?